prow/README.md

```diff
@@ -68,7 +169,9 @@ These values have been removed from the original
 [starter.yaml](https://github.com/kubernetes/test-infra/blob/master/prow/cluster/starter.yaml):
 
 - The `ConfigMap` values `plugins` and `config` because they are generated from
-  [config.yaml](config.yaml) and [plugins.yaml](plugins.yaml)
+  [config.yaml](config.yaml) and [plugin
```

Comment:
Did the line get cut here intentionally?  Looks like `# Deploy boskos` was accidentally pasted into the plugin link markdown?

Reply:
whoooooops...